app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/navigation/ChangeAddressGraph.kt

```diff
@@ -49,9 +52,27 @@ fun NavGraphBuilder.changeAddressGraph(
         navController = navController,
         backStackEntry = navBackStackEntry,
       )
+      BackHandler {
+        navController.popBackStack<ChangeAddressDestination.SelectHousingType>(inclusive = true)
```

Comment:
If the goal with this is to exit the entire moving flow completely, I *think* it's a good idea to do it on `AppDestination.ChangeAddress` instead, so the actual navigation graph route, and not the first destination. Because if the start destination does change for whatever reason this would stop working.

This is only if I understood correctly that this should exit the entire flow of course.